gcp_variant_transforms/libs/processed_variant.py

```diff
@@ -200,7 +201,8 @@ def __init__(
       use_allele_num=False,  # type: bool
       minimal_match=False,  # type: bool
       infer_annotation_types=False,  # type: bool
-      counter_factory=None  # type: metrics_util.CounterFactoryInterface
+      counter_factory=None,  # type: metrics_util.CounterFactoryInterface,
```

Comment:
nit: I prefer to switch the order of `counter_factory` and `allow_alternate_allele_info_mismatch`, so that all bool flags can stay together to improve the readability.

Reply:
Great suggestion! I actually moved it all the way up to be close to `split_alternate_allele_info_fields` as they're related.